apps/web/src/pages/Profile/CareerTimelineAndRecruitmentPage/constants.ts

```diff
@@ -1,23 +1,23 @@
 import { defineMessages } from "react-intl";
 
 export const PAGE_SECTION_ID = {
-  MANAGE_YOUR_RESUME: "manage-your-resume",
+  MANAGE_YOUR_CAREER_TIMELINE: "manage-your-career-timeline",
   QUALIFIED_RECRUITMENT_PROCESSES: "qualified-recruitment-processes",
 } as const;
 
 type ObjectValues<T> = T[keyof T];
 export type PageSectionId = ObjectValues<typeof PAGE_SECTION_ID>;
 
 export const titles = defineMessages({
-  resumeAndRecruitment: {
-    defaultMessage: "Résumé and recruitment",
-    id: "gDBVKb",
-    description: "Name of Résumé and recruitment page",
+  careerTimelineAndRecruitment: {
+    defaultMessage: "Career timeline and recruitment",
+    id: "Icl1fF",
+    description: "Name of Career timeline and recruitment page",
```

Comment:
Not related to this issue but should recruitment be pluralized? 🤔 It sounds reads a little odd to me being singular since it contains multiple recruitment processes in the list on this page.

Reply:
I guess it depends on if recruitment is meant to refer to:

1. the process of recruiting
2. recruitment processes

@gobyrne are you able to clarify?

Reply:
Let's pluralise everywhere. We list all the processes on this page, so I think plural makes more sense. It's already plural on the dashboard apparently.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/2494551/c5b4af11-b2b4-4853-8d97-373255e910d6)

Reply:
@gobyrne I don't think the word recruitment itself can be plural. If it refers to processes of recruitment, then the screenshot attached should really be: Career timeline and recruitment processes. Same goes for other references to "recruitments", in my opinion.

Reply:
Will raise discussion about Career timeline and recruitment vs. Career timeline and recruitments in next copy meeting and create an issue out of it if necessary.